src/libradosstriper/RadosStriperImpl.h

```diff
@@ -128,7 +128,7 @@ struct RadosStriperImpl {
   }
 
   // objectid manipulation
-  std::string getObjectId(const object_t& soid, long long unsigned objectno);
+  void getObjectId(const object_t& soid, uint64_t no, std::string& id);
```

Comment:
this can be RVO'ed. so no need to worry about the overhead of copy ctor.

Reply:
@tchaikov That's true. Compiler is *much* clever than me...